src/locale/kg.js

```diff
@@ -0,0 +1,78 @@
+//! moment.js locale configuration
+
+;(function (global, factory) {
+    typeof exports === 'object' && typeof module !== 'undefined'
+        && typeof require === 'function' ? factory(require('../moment')) :
+    typeof define === 'function' && define.amd ? define(['../moment'], factory) :
+    factory(global.moment)
+ }(this, (function (moment) { 'use strict';
```

Comment:
If you have a look at other locale files, you'll see they don't have the UMD header. It is autogenerated.